modules/installation-about-restricted-network.adoc

```diff
@@ -5,7 +5,10 @@
 // * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
 // * installing/installing_openstack/installing-openstack-installer-restricted.adoc
 // * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-ibm-power.adoc
```

Comment:
Please remove these added assembly references for IBM Z. They do not include this module on restricted networks.